opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableIndicators.jsx

```diff
@@ -112,15 +116,10 @@ class StixCyberObservableIndicatorsComponent extends Component {
       displayPromoteStix: false,
       displayCreate: false,
       promotingStix: false,
-      deleted: [],
     };
     this.indicatorParams = { first: 200 };
   }
 
-  onDelete(id) {
```

Comment:
Possible to remove the class component and refactor it to a pure function/typescript one ?